Content.Server/Botany/Components/PlantHolderComponent.cs

```diff
@@ -374,6 +389,13 @@ public void Update()
 
             CheckLevelSanity();
 
+            if (Seed.Sentient)
+            {
+                var comp = _entMan.EnsureComponent<GhostTakeoverAvailableComponent>(Owner);
```

Comment:
so based

Reply:
Thank you for the careful review and thoughtful feedback. I'll update the PR to address the issues I responded to above and wait for feedback on the others.